daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -320,6 +321,46 @@ packagingTests = testGroup "packaging"
               statusCode (responseStatus createResponse) @?= 200
               -- waitForProcess' will block on Windows so we explicitly kill the process.
               terminateProcess startPh
+    , testCase "daml start --sandbox-port=0" $ withTempDir $ \tmpDir -> do
+          writeFileUTF8 (tmpDir </> "daml.yaml") $ unlines
+              [ "sdk-version: " <> sdkVersion
+              , "name: sandbox-options"
+              , "version: \"1.0\""
+              , "source: ."
+              , "dependencies:"
+              , "  - daml-prim"
+              , "  - daml-stdlib"
+              , "start-navigator: false"
+              ]
+          let startProc = shell $ unwords
+                  [ "daml"
+                  , "start"
+                  , "--sandbox-port=0"
```

Comment:
So to clarify, this does not create a sandbox port file but does find a free port and pass that as the `ledger-port` to the JSON API?

Reply:
The assistant uses the --port-file option internally so it does technically create a portfile but not one visible to the user.

Reply:
Ok. In my testing the sandbox does not remove its port file on termination (a bug I think), so I'm wondering if that file will be visible to the user and if we need to clean it up.

Reply:
daml-helper creates a temp file and cleans it up itself.